tutorials/nlp/PTune_Sentiment_Analysis.ipynb

```diff
@@ -0,0 +1,686 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b7a434f4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BRANCH='main'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "developmental-gibraltar",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "You can run either this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+    "\n",
+    "Instructions for setting up Colab are as follows:\n",
+    "1. Open a new Python 3 notebook.\n",
+    "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+    "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+    "4. Run this cell to set up dependencies.\n",
+    "\"\"\"\n",
+    "# If you're using Google Colab and not running locally, run this cell\n",
+    "\n",
+    "# install NeMo\n",
+    "!python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[nlp]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "challenging-pioneer",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nemo.collections import nlp as nemo_nlp\n",
+    "from nemo.utils.exp_manager import exp_manager\n",
+    "\n",
+    "import os\n",
+    "import wget \n",
+    "import torch\n",
+    "import pytorch_lightning as pl\n",
+    "from omegaconf import OmegaConf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "employed-ethiopia",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we are going to describe how to use [P-Tuning method](https://arxiv.org/pdf/2103.10385.pdf) to find good prompts for large GPT models, so it can solve downstream NLP tasks with good performance. P-Tuning leverages few continuous free parameters to serve as prompts fed as the input to the pre-trained language models. Freezing the large language model weights, P-Tuning model can be trained efficiently while delivering stats of art performance. \n",
+    "\n",
+    "Large Language Model can be trained with [Megatron-LM project](https://github.com/NVIDIA/Megatron-LM), up to multi-billion parameters. In this notebook, we will use the pre-trained 344M GPT model released from NGC.\n",
```

Comment:
Can we recommend using NeMo for training?

Reply:
fixed the text to recommend NeMo.